e2e/cloud_manager/list_project_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e
+
+package cloud_manager
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+)
+
+func TestAtlasIAMProjects(t *testing.T) {
```

Comment:
This file is only for testing that evergreen is working as expected. I will remove this file once we start to implement e2e test for cloud manager

Reply:
what does this mean? do you plan to remove this later on? from a quick review of evergreen I think you could use the same `make e2e-test-iam` given that everything is controlled by environment variables so those changing should make your test "just work(?)"

Reply:
yes, I was thinking to remove this later on.